src/main/java/org/folio/rest/util/ResponseUtils.java

```diff
@@ -53,6 +53,7 @@ public static <T> Future<T> handleFailure(Throwable cause) {
 
   public static <T, V> void handleFailure(Promise<T> promise, AsyncResult<V> reply) {
     Throwable cause = reply.cause();
+    logger.error(cause.getMessage());
```

Comment:
This message can be duplicated if `badRequestMessage == null` (see later in the method).